engine/src/test/java/io/camunda/zeebe/process/test/engine/GrpcToLogStreamGatewayTest.java

```diff
@@ -22,7 +22,7 @@
 
 class GrpcToLogStreamGatewayTest {
 
-  static final List<String> UNSUPPORTED_METHODS = List.of("streamActivatedJobs");
+  static final List<String> UNSUPPORTED_METHODS = List.of();
```

Comment:
:thought_balloon: I figured we might as well keep the mechanism, even if we now support all methods.